autokeras/auto_model.py

```diff
@@ -24,27 +24,31 @@ class AutoModel(kerastuner.HyperModel):
     The user can specify the inputs and outputs of the AutoModel. It will infer
     the rest of the high-level neural architecture.
 
-    Attributes:
+    # Arguments
         inputs: A list of or a HyperNode instance.
             The input node(s) of the AutoModel.
         outputs: A list of or a HyperHead instance.
             The output head(s) of the AutoModel.
         max_trials: Int. The maximum number of different models to try.
+            Defaults to 100.
         directory: String. The path to a directory for storing the search outputs.
+            Defaults to './tmp'.
+        seed: Int. Random seed.
     """
 
     def __init__(self,
                  inputs,
                  outputs,
-                 max_trials=None,
-                 directory=None,
-                 **kwargs):
-        super().__init__(**kwargs)
+                 max_trials=100,
+                 directory='./tmp',
```

Comment:
In Keras Tuner it defaults to the current directory (where a new subfolder for the current project gets created).